packs/data/feats.db/wellspring-mage-dedication.json

```diff
@@ -13,7 +13,7 @@
             "value": null
         },
         "description": {
-            "value": "<p>There's no longer a limit on how many temporary spell slots you can gain per day from wellspring magic.</p>\n<p><strong>Special</strong> You can't select another dedication feat until you gain two other feats from the wellspring mage archetype.</p>"
+            "value": "<p>There's no longer a limit on how many temporary spell slots you can gain per day from wellspring magic.</p>\n<p><strong>Special</strong> You can't select another dedication feat until you gain two other feats from the @UUID[Compendium.pf2e.journals.vx5FGEG34AxI2dow.JournalEntryPage.zj9z2BwVX6TLHdx3]{Wellspring Mage (Class Archetype)} archetype.</p>"
```

Comment:
Can ditch the parenthetical in the label.

Reply:
Check.